app/controllers/registrar/domain_transfers_controller.rb

```diff
@@ -24,8 +24,8 @@ def create
         request.basic_auth(current_registrar_user.username,
                            current_registrar_user.plain_text_password)
 
-
-        response = do_request(request, uri)
+        action = Actions::DoRequest.new(request, uri)
+        response = action.call
```

Comment:
BTW what will happen if request somehow fails?

Don't see any rescues or error processing, suppose we might add some.